### ISSUE TITLE
UnifiedAlchemyMagicMock does not work with .label()

### ISSUE THREAD

**Issue by hozher** [miki725/alchemy-mock#24]
Hello, 

I have this query in my function to be tested:
```python
query = session.query(
            whitelist.whitelistEmailId,
            whitelist.areaId,
            whitelist.businessNameId,
            whitelist.email,
            whitelist.identificationNumber,
            whitelist.isRegistered,
            whitelist.serviceTypeId,
            area.name.label('area'),
            business.name.label('businessName'),
            service.name.label('serviceType')
            ).\
        join(area, whitelist.areaId == area.areaId).\
        join(business, whitelist.businessNameId == business.businessNameId).\
        join(service, whitelist.serviceTypeId == service.serviceTypeId)
```
I am trying to mock it out like this:
```python
return UnifiedAlchemyMagicMock(data=[
        (
            [mock.call.query(
                WhitelistedEmail.whitelistEmailId,
                WhitelistedEmail.areaId,
                WhitelistedEmail.businessNameId,
                WhitelistedEmail.email,
                WhitelistedEmail.identificationNumber,
                WhitelistedEmail.isRegistered,
                WhitelistedEmail.serviceTypeId,
                Area.name.label('area'),
                BusinessName.name.label('businessName'),
                ServiceType.name.label('serviceType')
            )], Factory.get_list()
        )
    ])
```
After Ive added these three guys, that make use of .label()  does not work anymore.
```python
    Area.name.label('area'),
    BusinessName.name.label('businessName'),
    ServiceType.name.label('serviceType')
```

Is this a known issue? or might be am I doing something wrong. any clue is more than welcome.

**Comment by miki725** [comment]
dont think tested with labels. could be the comparison logic does not correctly compare labels. Can you try using https://github.com/miki725/alchemy-mock/blob/e8a9d6136de4af7ea80da8bc493653d83b7ee0cb/alchemy_mock/comparison.py#L60 to manually compare labels?
 so something like:

```python
ExpressionMatcher(Area.name.label('area')) == Area.name.label('area')
```

**Comment by hozher** [comment]
@miki725 Thanks for answering.

The problem is that when `query` gets called is not returning any value (not getting mocked) I did the `ExpressionMatcher` whiteout success as follows:
```python
return UnifiedAlchemyMagicMock(data=[
        (
            [mock.call.query(
                WhitelistedEmail.whitelistEmailId,
                WhitelistedEmail.areaId,
                WhitelistedEmail.businessNameId,
                WhitelistedEmail.email,
                WhitelistedEmail.identificationNumber,
                WhitelistedEmail.isRegistered,
                WhitelistedEmail.serviceTypeId,
                ExpressionMatcher(Area.name.label('area')) == Area.name.label('area'),
                ExpressionMatcher(BusinessName.name.label('businessName')) == BusinessName.name.label('businessName'),
                ExpressionMatcher(ServiceType.name.label('serviceType')) == ServiceType.name.label('serviceType'),
            )], Factory.get_list()
        )
    ])
```

**Comment by miki725** [comment]
I meant can you post results running `ExpressionMatcher` by itself. I dont know if its capable of comparing label objects

**Comment by hozher** [comment]
alright, i got this:

```python
 print(ExpressionMatcher(Area.name.label('area')) == Area.name.label('area'))
    print(ExpressionMatcher(BusinessName.name.label('businessName')) == BusinessName.name.label('businessName'))
    print(ExpressionMatcher(ServiceType.name.label('serviceType')) == ServiceType.name.label('serviceType'))
```

```bash
"Area".name = "Area".name
"BusinessName".name = "BusinessName".name
"ServiceType".name = "ServiceType".name
```

**Comment by miki725** [comment]
yep thats not supported. should of gotten a boolean if it was. Ill try to add support at some point or feel free to submit a PR!

**Comment by miki725** [comment]
https://github.com/miki725/alchemy-mock/pull/25 adds labels support. can you test that branch?

**Comment by hozher** [comment]
It works pretty well 😃 

```bash
>>> from alchemy_mock.comparison import ExpressionMatcher
>>> from model import Area
>>> ExpressionMatcher(Area.name.label('area')) == Area.name.label('area')
True
```

Thanks a lot!

**Comment by miki725** [comment]
just released `0.4.2`. should be able to install from pypi. let me know if any further issues
